app/model/commands/PublishAtomCommand.scala

```diff
@@ -141,11 +147,41 @@ case class PublishAtomCommand(id: String, override val stores: DataStores, youTu
     }
   }
 
+  private def parseDescriptionForYoutube(description: String): String = {
+      val html = Jsoup.parse(description)
+      html.select("a").remove()
```

Comment:
All the html tags are stripped in the line below (`html.text()`) so all the list tags will be stripped away. This line removes all links with their content so in the example you shared, all the contents on the list would be removed. If the list contained text instead of a link, the text inside would remain but the list tags would be removed. I guess it would be simpler to always remove all lists, not sure if there is a case when lists are used without links.